nbproject/nbjdk.properties

```diff
@@ -0,0 +1 @@
+nbjdk.active=JDK_1.8
```

Comment:
This file sets the active JDK for this project. I'm not 100% certain it should be included.